### PR TITLE
User Story 8 - Search Bar

### PR DESCRIPTION
**Pull request by kevinldg**
Searchbar for searching through the upcycling ideas via hashtags and titles.

Corresponding user story: https://github.com/kevinldg/upcycling-app/issues/4

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -35,22 +40,81 @@ export default function HomePage({
...
+  const [searchValue, setSearchValue] = useState("");
+
+  const fuse = new Fuse(defaultIdeas, {
+    keys: ["hashtags", "title"],
+  });
+
+  function handleClickEvent(value) {
+    setSearchValue(value);
+    setSearchResults(fuse.search(value));
+    console.log(searchValue);
```
Comment (CJAkkuec) [inline pages/index.js:56]:
```suggestion
```
img

File: pages/index.js [hunk 2]
```diff
@@ -35,22 +40,81 @@ export default function HomePage({
...
+              isFavourite={searchResult.item.isFavourite}
+              id={searchResult.item.id}
             />
-            <StyledLink href={`/ideaDetails/${idea.id}`}>See More</StyledLink>
+            <StyledLink href={`/ideaDetails/${searchResult.item.id}`}>
+              See More
+            </StyledLink>
           </CardListItem>
-        ))}
+        ))} */}
```
Comment (CJAkkuec) [inline pages/index.js:117]: **No commented out code!** 📣

**Review by CJAkkuec — APPROVED** [review]
Approved, but with two **VERY IMPORTANT** notes.

**Comment by kevinldg** [comment]
Thx for your feedback @CJAkkuec ! We've implemented your suggested changes (directly pushed into main after merge) 👍

Commit: https://github.com/kevinldg/upcycling-app/commit/3bdfc7e9ac3875c81612657a3e6081f8da23ab23

**Pull request merged**